freeze.py

```diff
@@ -347,7 +347,10 @@ def find_files(directory, patterns):
     for filename in find_files(os.path.join(qt_webengine_path, "Resources", "qtwebengine_locales"), ["*"]):
         external_so_files.append((filename, os.path.relpath(filename, start=os.path.join(qt_webengine_path, "Resources"))))
     for filename in find_files(os.path.join(qt_install_path, "plugins"), ["*"]):
-        external_so_files.append((filename, os.path.relpath(filename, start=os.path.join(qt_install_path, "plugins"))))
+        relative_filepath = os.path.relpath(filename, start=os.path.join(qt_install_path, "plugins"))
+        plugin_name = os.path.split(relative_filepath)[0]
```

Comment:
This feels perennial, but... :wink: 
```suggestion
        plugin_name = os.path.basename(relative_filepath)
```

Reply:
Sorry, wait, no, I read that backwards.
```suggestion
        plugin_name = os.path.dirname(relative_filepath)
```

...Though, that seems odd to me.

Reply:
Oh, I see now, it's the dirname relative to the plugin directory.